app/client.go

```diff
@@ -2,38 +2,37 @@ package appunlynx
 
 import (
 	"os"
+	"regexp"
+	"strconv"
+	"strings"
 
 	"github.com/btcsuite/goleveldb/leveldb/errors"
 	"github.com/lca1/unlynx/lib"
-	"github.com/lca1/unlynx/services/default"
+	"github.com/lca1/unlynx/services"
 	"go.dedis.ch/onet/v3"
 	"go.dedis.ch/onet/v3/app"
 	"go.dedis.ch/onet/v3/log"
 	"gopkg.in/urfave/cli.v1"
-	"regexp"
-	"strconv"
-	"strings"
 )
 
 // BEGIN CLIENT: QUERIER ----------
 func startQuery(el *onet.Roster, proofs bool, sum []string, count bool, whereQueryValues []libunlynx.WhereQueryAttribute, predicate string, groupBy []string) {
-	client := servicesunlynxdefault.NewUnLynxClient(el.List[0], strconv.Itoa(0))
+	client := servicesunlynx.NewUnLynxClient(el.List[0], strconv.Itoa(0))
 
-	// Generate Survey Data
 	nbrDPs := make(map[string]int64)
 	//how many data providers for each server
 	for _, server := range el.List {
 		nbrDPs[server.String()] = 1 // 1 DP for each server
 	}
 
-	surveyID, err := client.SendSurveyCreationQuery(el, servicesunlynxdefault.SurveyID(""), nil, nbrDPs, proofs, true, sum, count, whereQueryValues, predicate, groupBy)
+	surveyID, err := client.SendSurveyCreationQuery(el, servicesunlynx.SurveyID(""), nil, nbrDPs, proofs, true, sum, count, whereQueryValues, predicate, groupBy)
 	if err != nil {
 		log.Fatal("Service did not start.", err)
 	}
 
 	grp, aggr, err := client.SendSurveyResultsQuery(*surveyID)
 	if err != nil {
-		log.Fatal("Service could not output the results.")
+		log.Fatal("Service could not output the results.", err)
```

Comment:
Is this really `Fatal`, or should this just return the error? `Fatal` really means: "the internal state of the conode is inconsistent and every further action might make it worse". While here I think it should only be: "oups, something went wrong. Let's discard what happened and wait for a new request."

Reply:
Same thing for all `Fatal`s in the rest of the code...

Reply:
I am checking all `log.Fatal`s now. I will add a commit after reviewing all of them

Reply:
@ineiti just commited this 5ad4d7e it should fix this comment (just of the app part)